ReactSkia/sdk/OnScreenKeyBoard.cpp

```diff
@@ -592,13 +592,15 @@ void OnScreenKeyboard::createOSKLayout(OSKTypes oskType) {
       oskLayout_.siblingInfo=&symbolKBKBKeySiblingInfo;
       oskLayout_.kbGroupConfig=symbolKBGroupConfig;
       oskLayout_.returnKeyIndex=symbolKBReturnKeyIndex;
+      oskLayout_.defaultFocussIndex=numericKBDefaultHLKeyIndex;
```

Comment:
this should be symbolKBDefaultHLKeyIndex?

Reply:
Yes, Copy/Paste Error. Nicely caught.